shiny/ui/_bootstrap.py

```diff
@@ -47,7 +47,7 @@ def row(*args: TagChild | TagAttrs, **kwargs: TagAttrValue) -> Tag:
     elements that should appear on the same line (if the browser has adequate width) and
     :func:`~shiny.ui.column` to define how much horizontal space within a 12-unit wide
     grid each on of these elements should occupy. See the [layout
-    guide](https://shiny.rstudio.com/articles/layout-guide.html>) for more context and
+    guide](https://shiny.posit.co/articles/layout-guide.html>) for more context and
```

Comment:
This article is about R. Is it accurate enough that it's worth linking to here, noting that the principles are the same but the function names have to be adapted to Python? Is there something better to link here? Or should we drop it? 

cc @cpsievert